src/EFCore.MySql/MySqlDbFunctionsExtensions.cs

```diff
@@ -450,5 +450,54 @@ public static class MySqlDbFunctionsExtensions
             => (startDate.HasValue && endDate.HasValue)
                 ? (int?)DateDiffMicrosecond(_, startDate.Value, endDate.Value)
                 : null;
+
+        /// <summary>
+        ///     <para>
+        ///         An implementation of the SQL LIKE operation. On relational databases this is usually directly
+        ///         translated to SQL.
+        ///     </para>
+        ///     <para>
+        ///         Note that if this function is translated into SQL, then the semantics of the comparison will
+        ///         depend on the database configuration. In particular, it may be either case-sensitive or
+        ///         case-insensitive. If this function is evaluated on the client, then it will always use
+        ///         a case-insensitive comparison.
+        ///     </para>
+        /// </summary>
+        /// <param name="_">The DbFunctions instance.</param>
+        /// <param name="matchExpression">The string that is to be matched.</param>
```

Comment:
This comment seems wrong; `matchExpression` is not a `string`.

Reply:
Sorry,I've fixed it.